Code/GraphMol/FilterCatalog/FilterCatalog.h

```diff
@@ -129,11 +129,12 @@ class RDKIT_FILTERCATALOG_EXPORT FilterCatalog : public FCatalog {
 
   FilterCatalog(FilterCatalogParams::FilterCatalogs catalogs)
       : FCatalog(), d_entries() {
-    setCatalogParams(new paramType_t(catalogs));
+    auto temp_params = static_cast<paramType_t>(catalogs);
```

Comment:
I shouldn't have been, but I was surprised that this works.
I still find it less than clear and would prefer:
```suggestion
    paramType_t temp_params(catalogs);
```

Reply:
Ok.